platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/AccountsManagerTest.java

```diff
@@ -15,10 +23,37 @@ public void testIsExpired() {
     long futureValue = now + 1;
     long immediatePast = now - 1;
 
-    Assert.assertTrue(AccountsManager.isExpired(now, defaultValue));
-    Assert.assertTrue(AccountsManager.isExpired(now, tooOld));
+    assertTrue(AccountsManager.isExpired(now, defaultValue));
+    assertTrue(AccountsManager.isExpired(now, tooOld));
+
+    assertFalse(AccountsManager.isExpired(now, futureValue));
+    assertFalse(AccountsManager.isExpired(now, immediatePast));
+  }
+
+  @Test
+  public void checksSkuTokenExternalManagement() {
+    SharedPreferences mockedSharedPreferences = mock(SharedPreferences.class);
+    when(mockedSharedPreferences.getString(KEY_PREFERENCE_SKU_TOKEN, "")).thenReturn("external-sku-token");
+    boolean isNotManaged = false;
```

Comment:
That was on purpose for better readability when creating the `AccountsManager` object https://github.com/mapbox/mapbox-gl-native/blob/434eacd60cd8bdf933894c6bfe9b4fe32ad560c5/platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/AccountsManagerTest.java#L38

Reply:
:thinking: now the variable is called `isNotManaged`, and has value `false`, so that should mean that it _is_ managed. Idk, just seems a little bit off to me.

Reply:
👀 [_Working Effectively with Unit Tests_](https://leanpub.com/wewut/read) 
> when writing tests you should prefer DAMP (Descriptive And Maintainable Procedures) 

That includes a bunch of _procedures_ [Moving Towards Readability](https://leanpub.com/wewut/read#leanpub-auto-moving-towards-readability) 😅 

I highly recommend you reading that book (if you haven't already). It's 🔝 💯

Reply:
Thanks! It had a reverse effect on me though haha :man_shrugging: